docs/outreach-api.md

```diff
@@ -52,11 +52,11 @@ Outreach API access:
    function
 
    - Addon opens popup using the Outreach [API authentication URL](https://api.outreach.io/api/v2/docs#authentication).
-     Constructing of this url relies on manifest.api section configuration: [applicationId](manifest.md#applicationid),
-     [redirectUri](manifest.md#redirecturi) and [scopes](manifest.md#scopes) values.
+     Constructing of this url relies on manifest.api section configuration: [client.id](manifest.md#client),
+     [redirectUris](manifest.md#redirecturis) and [scopes](manifest.md#scopes) values.
    - When user clicks **Authorize** button on [Outreach consent screen](assets/api-consent.png), Outreach will redirect
-     to `/authorize` endpoint on ([manifest.api.redirectUri](manifest.md#redirecturi)) address with a short living
-     authorization code passed as query param.
+     to `/authorize` endpoint on one of ([manifest.api.redirectUris](manifest.md#redirecturis)) address with a short
```

Comment:
What you mean by "one of"? Is there some precise rule which URL is taken? I.e. tries first URL with 2s timeout then tries the next, etc. ? Or does it call random one?

Reply:
I see now from documentation below.